packages/circuit-ui/components/Table/components/TableCell/TableCell.ts

```diff
@@ -131,10 +127,6 @@ const TableCell = styled.td<TableCellProps>`
   ${condensedPresentationStyles};
 `;
 
-/**
- * FIXME: we don't need this data-testid (should query by role when testing)
- * but some might rely on it in their tests, so this will be a breaking change.
- */
 TableCell.defaultProps = {
   'data-testid': 'table-cell',
 };
```

Comment:
What's the reason for leaving this test id?

Reply:
I actually removed it in a94f6a3, and then added it back in 896e3f8.

My reasoning behind it was that we still use testids in specs for other Table components (TableHeader, TableRow), so it's a bit inconsistent to remove it here. Happy to do it if you think it makes more sense 👍

Reply:
I'd like to remove all default props whenever possible, so if there's an alternative way to pass test ids in the specs, I'd switch to that.

Reply:
I'll try to remove all testids then, I guess I can update the specs to query by table roles just like I did with the table cells

Reply:
Done in 202d130, replaced in tests by `getByRole` queries